tox.ini

```diff
@@ -0,0 +1,68 @@
+[tox]
+envlist = py{35,36,37}-{pytest,setup}{,-all},docs,dev,flake8
+skipsdist = True
+requires = tox-venv
+           setuptools >= 30.0.0
+
+[testenv]
+usedevelop = True
+whitelist_externals = which
+passenv = HOME LC_ALL LC_CTYPE
```

Comment:
It's also worth passing `CC` here so that people who are using ccache etc can still use it with tox.

Reply:
Yep, I added CC and CFLAGS, which should fix the coverage for C files.